src/cascadia/WindowsTerminalUniversal/Package-Dev.appxmanifest

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" 
```

Comment:
>Package [](start = 1, length = 7)

If you want unification of deployment and settings for both UWP mode and Win32 mode, I sugguest to move these files into the TerminalApp project and include the Toolkit Win32 SDK targets project file.

Reply:
that put it in a subfolder, as we found out, which would make it subject to weird rewrite rules or have us do something even more complicated overriding the additional path. The subfolder was because it was a lib, not an exe. Just sticking with what we have and cleaning it up.